GetIntoTeachingApi/Controllers/TeachingEventsController.cs

```diff
@@ -91,6 +91,22 @@ public async Task<IActionResult> Get([FromRoute, SwaggerParameter("The `readable
             return Ok(teachingEvent);
         }
 
+        [HttpGet]
+        [CrmETag]
+        [PrivateShortTermResponseCache]
+        [Route("teaching_event_buildings")]
+        [SwaggerOperation(
+         Summary = "Retrieves all event buildings.",
+         OperationId = "GetTeachingEvent",
+         Tags = new[] { "Teaching Events" })]
+        [ProducesResponseType(typeof(TeachingEventBuilding), 200)]
+        public IActionResult GetTeachingEventBuildings()

```

Comment:
As we're going to have a couple of endpoints here (there will be another for creating a building) I would be inclined to put this in its own `TeachingEventBuildingsController`. Also typo in the `OperationId` should be `GetTeachingEventBuildings`.